src/utils/createError.ts

```diff
@@ -0,0 +1,11 @@
+/** Create Error from error of unknown type */
+
+export function createError(error: unknown): Error {
```

Comment:
This is basically just a Model class, no? I'm fine with it being a `util` thing as well. I think in certain edge cases you get back full HTML pages from broken requests as well, we might want to test this against that scenario.

Reply:
The `createError` util has only one use case. In `try catch` blocks, the caught error is of unknown type and my editor was complaining I can't set the error.message on an unknown type, so I had to convert to an `Error`. I'll add a comment to the util description.